README.md

```diff
@@ -39,37 +36,129 @@ Figure by: Kristoffer Rakstad Solberg
 
 1. Enter the folder where you want to clone the repostory:
 	```bash
-	cd manta_ws/src
+	cd vortex_ws/src
 	```
 
 2. Clone the repository: 
 	```bash
-	git clone https://github.com/vortexntnu/manta-auv-simulator.git
+	git clone https://github.com/vortexntnu/vortex-simulator.git
 	```
 
 3. Compile the code by running "catkin build" inside the workspace:
 	```bash
-	cd ~/manta_ws/
+	cd ~/vortex_ws/
 	catkin build
 	```
-## 2. Run Manta V1 in Simulation with Gazebo, Smach viewer, Camera pop-up windows etc ##
+## 2. Run Vortex Simulator ##
 -------------------------
 
-1. Open a new terminal and run your simulation world. This will upload Manta (w/ sensor, camera, thrusters etc.) and launch robot localization as well. i.e :
+### Method 1. Using launch files ### 
+
+1. Open a new terminal and launch a simulation. This will upload Manta (w/ sensor, camera, thrusters etc.), start the simulation world, upload any other object models, and launch robot localization i.e :
 	```bash
-	roslaunch simulator_launch cybernetics_pool.launch
+	roslaunch simulator_launch cybernetics_sim.launch
 	```
-
+Builtin simulator worlds:
+- basin_sim.launch
+- cybernetics_sim.launch
+- robosub_sim.launch
+- vortex_sim.launch
+
+ROS launch arguments:
```

Comment:
I really like the look of this, and it imakes the new features very easy to use. Since a lot of new and useful features have been added, it would be nice to add this to the readme in the main repo. Once I get around to it I will add this to the wiki as well, in the User Guide section.